shuffle/genesis/src/main.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use std::path::Path;
+use structopt::StructOpt;
+use diem_config::config::NodeConfig;
+
+#[derive(StructOpt)]
+#[structopt(
+    name = "custom-node",
+    about = "Builds a WriteSet transaction to install the custom modules and starts a node",
+    rename_all = "kebab-case"
+)]
+pub struct CustomFramework {
+    /// Directory where the node config will be generated. Must not already exist
+    #[structopt(long = "node-config-dir")]
+    node_config_dir: String,
+}
+/// Generate a node config under `args.node_config_dir`
+fn main() -> Result<()> {
```

Comment:
Could we add an option to the node startup command to be able to configure this open publishing flag?  https://github.com/diem/diem/blob/main/diem-node/src/main.rs#L39

Reply:
Added. Now running with `--open-publishing` will turn the option on.